s3/compress.go

```diff
@@ -481,3 +485,8 @@ func (mc *MockStorageHandler) ProcessSources(db models.DBInterface, uid uuid.UUI
 
 	fmt.Println("Ran mockStorageHandler.ProcessSources")
 }
+
+func (mc *MockStorageHandlerLarge) CreateObject(ctx context.Context, l *zap.SugaredLogger, db models.DBInterface, body io.Reader, application string, resourceUUID uuid.UUID, payload *models.ExportPayload) error {
+	fmt.Println("Ran mockStorageHandler.CreateObject")
+	return &http.MaxBytesError{}
+}
```

Comment:
Instead of making a new mock, can you try just reading the contents of the `io.Reader` in the original mock?  Seems like you should be able to just read the data, throw it away and return an error if an error is returned by the reader.